### DISCUSSION TITLE
Combine Kimai with hardware time clock?

### DISCUSSION THREAD

**Discussion by MrReg99** [kimai/kimai#5119] [General]
Hello, I am new to Kimai but a friend of my highly recommended it as he is using the software since years for his private school to track the working times of the teachers.

However, my request is a bit different; I tested a bit and like the system but basically I a hardware time clock (in combination with Kimbai): 3 Offices, 20 staff member. They all of NFC keys for the door locks and ideally they enter the office in the morning and check in with the system and check out in the evening and it gets recorded as working time and me (the boss) can see reports about this.

Is this possible with Kimai?

Regards, R

**Comment by kevinpapst** [comment]
Hey 👋 

There is the Kiosk mode: https://www.kimai.org/store/kiosk-barcode-bundle.html

You can combine it with a hardware of your choice. 
There are good Android tablets out there, which also support NFC readers.
Install a "Kiosk browser" app and point it to Kimai.

Best regards
Kevin

**Comment by MrReg99** [comment]
Thank you, Kevin! I will do some research in that direction!
